plugin/orca.vim

```diff
@@ -201,6 +216,7 @@ function! s:setup_dstatus()
     setlocal bufhidden=delete
     setlocal nowrap
     set filetype=dstatus
+    nmap <buffer> k :call <SID>DockerKill(<SID>line_columns([0])[0])<CR>r
```

Comment:
Change this to `K` to prevent accidentally killing containers when trying to navigate up the list of running containers.